chat-client-ui-types/src/uiContracts.ts

```diff
@@ -32,6 +34,8 @@ export type UiMessageParams =
     | AuthFollowUpClickedParams
     | GenericCommandParams
     | ErrorParams
+    | SendToPromptParams
+    | QuickActionsOptions
```

Comment:
Shouldn't chat client receive the whole `initializeResult` from server?

Reply:
Whole `initializeResult` can contain too many not relevant data. 

It maybe better to specify more generic wrapping initialise object expected by chat client instead of `QuickActionsOptions` specifically. This way this object can be extended. What happens if tomorrow we need to extend this to more data?